### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by augustoproiete** [augustoproiete/serilog-sinks-richtextbox#36] [Announcements]
### :zap: New features / improvements

- #12 Implement async batching via queue (@Skaptor)

Pull requests were reviewed by: @augustoproiete, @TonyValenti

---

Supported features and usage examples are described in the [README](https://github.com/serilog-contrib/serilog-sinks-richtextbox/blob/master/README.md).

The full list of changes is available [here](https://github.com/serilog-contrib/serilog-sinks-richtextbox/milestone/3?closed=1).
NuGet package available on [nuget.org](https://www.nuget.org/packages/Serilog.Sinks.RichTextBox.Wpf/1.1.0).

<hr /><em>This discussion was created from the release <a href='https://github.com/serilog-contrib/serilog-sinks-richtextbox/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by TonyValenti** [comment]
@augustoproiete - This release will cause performance issues because:
1.  It create a thread per logger
2. The threads spin while waiting for log data
3. The threads are never disposed.
